code/modules/surgery/tools.dm

```diff
@@ -113,14 +113,8 @@
 		return
 	if(isorgan(I))

```

Comment:
No idea, I don't understand code like I said. Do I remove Lines 115 through to 118?